doc/source/contribution/documentation.md

```diff
@@ -49,6 +45,19 @@ Clone Mroonga repository to working directory. Don't forget to do "Git configura
 % git remote add upstream git@github.com:mroonga/mroonga
 ```
 
+### Required softwares
+
+Generating Mroonga documentations needs [Sphinx](https://www.sphinx-doc.org/) and MySQL's source and build directories.
+Instructions for installing Sphinx are provided below.
+Please see {doc}`/install/others` for information on preparing the MySQL source and build directories.
```

Comment:
You might want to introduce how to install Groonga from source with link to https://groonga.org/docs/install/others.html#build-from-source.

Reply:
I think it's good idea.
How about adding it to https://mroonga.org/ja/docs/install/others.html#libraries page when I edit at the following issue.